avocado/core/nrunner/runnable.py

```diff
@@ -267,9 +270,16 @@ def get_capabilities_from_runner_command(runner_command, env=None):
         out, _ = process.communicate()
 
         try:
-            return json.loads(out.decode())
+            capabilities = json.loads(out.decode())
         except json.decoder.JSONDecodeError:
-            return {}
+            capabilities = {}
+
+        # lists are not hashable, and here it'd make more sense to have
```

Comment:
I would move filling of `STANDALONE_EXECUTABLE_CONFIG_USED` in its own method, or I would update the method docstring to inform about `STANDALONE_EXECUTABLE_CONFIG_USED` manipulation.